src/resources/swagger.yaml

```diff
@@ -389,6 +440,18 @@ paths:
 
 servers:
   - url: /
+tags:
+  - name: Health
+    description: Service status monitoring
+  - name: Schemas
+    description: View and manage the Data Schema used by Argo-Clinical
+  - name: Registration
+    description: Registering new Donors, Specimens, and Samples
+  - name: Submission
+    description: Uploading, Validating, and Committing clinical data
+  - name: Clinical Data
+    description:
```

Comment:
does it let you do empty strings for description?

Reply:
It doesn't break anyways... I didn't know what to write here, so this is just placeholder.